packages/cli-lib/lib/config.js

```diff
@@ -539,6 +543,39 @@ const updateDefaultAccount = defaultAccount => {
   writeConfig();
 };
 
+/**
+ * @throws {Error}
+ */
+const renameAccount = async (currentName, newName) => {
+  if (!newName) {
+    throw new Error('Please provide a new name for the portal');
+  }
```

Comment:
Not sure if I'm doing something wrong on my end but I wasn't able to hit this, I never got past yargs' argument check: `Not enough non-option arguments: got 1, need at least 2`

Reply:
Good call, I think we can remove this because yargs catches it.